go.mod

```diff
@@ -1,11 +1,5 @@
 module github.com/go-chi/httplog
```

Comment:
can you update this to: `github.com/go-chi/httplog/v2` since its a breaking change to the api, will be safer this way